src/function/scalar/string/length.cpp

```diff
@@ -37,6 +37,9 @@ void LengthFun::RegisterFunction(BuiltinFunctions &set) {
 	                               ScalarFunction::UnaryFunction<string_t, int64_t, StrLenOperator, true>));
 	set.AddFunction(ScalarFunction("bit_length", {SQLType::VARCHAR}, SQLType::BIGINT,
 	                               ScalarFunction::UnaryFunction<string_t, int64_t, BitLenOperator, true>));
+	// length for BLOB type
+	set.AddFunction(ScalarFunction("octet_length", {SQLType::VARCHAR}, SQLType::BIGINT,
```

Comment:
This is a BLOB function, right? If so, it should not take SQLType::VARCHAR as parameter, but it should take SQLType::BLOB.